stdlib/public/core/StringBridge.swift

```diff
@@ -281,9 +281,13 @@ extension String {
       }
     }
     if _guts._object.isImmortal {
+      // TODO: We'd rather emit a valid ObjC object statically than create a
```

Comment:
Do we have a bug tracking this? I should either CC on it or steal it if so

Reply:
If we do, then I assigned it to you ;-)